### ISSUE TITLE
Ryo from bank missing after Yokai Guard battle

### ISSUE THREAD

**Issue by RyuuTakeshi** [MathiasGruber/TheNinjaRPG#117]
User: RyuTakeshi

Description of the bug:  I did the quest Prelude to Momo, and I fought the second enemy. I was near him, and I think he tried to summon, them I got a red error that also said something "to fast, bla bla 1% of your ryo". I didn't give it much taught, as usually the red errors are harmless, but after the guard killed me I was missing about 300k+, I remember having near 800k

How to reproduce: I think that Yokai Guard caused the error because I was on the summoning spot where Momo should of appeared. I couldn't move, kept getting that error on every move I tried to do, and only after I used  Chakra echo I could see the summons and and got a lot of damage in a short time 

Expected behavior: Money still in the bank

Device information: when it happened I think I was on Iphone, IOS, Safari

**Comment by RyuuTakeshi** [comment]
UPDATE

Description of the bug: While engaged in the Battle Tower: Prelude to Momo, I encountered a critical issue. During the fight with the second mob, Yokai Guardian, the game lagged when the enemy’s health reached approximately 30%. This caused my character to become unresponsive. I then experienced an error, which I have since learned is associated with a 1% ryo penalty typically applied to auto-clickers. Despite never using an auto-clicker, I clicked multiple times to regain control, resulting in a loss of over 300k ryo.

Steps to Reproduce: Engage in a battle with the Yokai Guardian in the Prelude to Momo quest and observe for lag when Momo is summoned and the Yokai Guardian's health drops below 30%.

Expected Behavior: 
 The 1% ryo penalty should not be triggered for users who are not employing auto-clickers. I expect this issue to be addressed, and I respectfully request the restoration of my lost ryo. It is extremely frustrating to be penalized for an action I did not commit, particularly with no prior awareness of such a penalty.

Device Information:
- Device: iPhone  
- Operating System: iOS  
- Browser: Safari
